lib/rules/max-attributes-per-line.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @fileoverview Define the number of attributes allows per line
+ * @author Filipa Lacerda
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Define the number of attributes allows per line',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: null,
+    schema: [{
+      type: 'object',
+      properties: {
+        'singleline': {
+          type: 'number',
+          minimum: 1
+        },
+        'multiline': {
+          type: 'object',
+          properties: {
+            'max': {
+              type: 'number',
+              minimum: 1
+            },
+            'allowFirstLine': {
+              type: 'boolean'
+            }
+          }
+        }
+      }
+    }]
+  },
+
+  create: function (context) {
+    const configuration = context.options[0] || { singleline: 3, multiline: { max: 1, allowFirstLine: false }}
```

Comment:
Please update settings so it's possible to override only part of it without the need to update everything. You can use `Object.assign` to merge passed settings with default ones:
```
const configuration = Object.assign({}, context.options[0] || {}, {
  singleline: 3,
  multiline: { max: 1, allowFirstLine: false }
})
```

And I'd also consider accepting setting: `multiline: 4` as just a number, then we would assume `allowFirstLine` is false. What do you think?